rpc_jobs/rpc_gating.yml

```diff
@@ -0,0 +1,20 @@
+- job:
+    name: "Merge-Trigger-rpc-gating"
+    project-type: pipeline
+    concurrent: true
+    properties:
+      - rpc-gating-github
+      - build-discarder:
+          num-to-keep: 30
+    triggers:
+        - github
+    parameters:
+      - rpc_gating_params
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/rcbops/rpc-gating
+            branches:
+              - "${RPC_GATING_BRANCH}"
+            credentials-id: "github_account_rpc_jenkins_svc"
+      script-path: job_dsl/rpc_gating_merge_trigger.groovy
```

Comment:
@odyssey4me slack is still broken for me so I've only just seen the e-mail notification. Not had this before but based on the error, try using a dsl block and put the contents of rpc_gating_merge_trigger.groovy in it.

Reply:
Thanks, I worked it out. It turns out it comes down to where the script was running from and whether it had been imported correctly. I did some rework and it's working like a charm now.